### PR TITLE
Improve timeline of bubbles

### PR DESCRIPTION
**Pull request by basilesimon**
Include annotation and prettier code

screen shot 2018-05-18 at 18 07 59

screen shot 2018-05-18 at 18 07 35

### CODE REVIEW

File: timeline-bubbles/D3/script.js [hunk 1]
```diff
@@ -19,9 +23,10 @@ const margin = { top: 40, right: 100, bottom: 30, left: 20 },
     margin.top -
     margin.bottom;
 
-// Clean up before drawing
-// By brutally emptying all HTML from plot container div
-d3.select('#times-timeline').html('');
+(config.yTranslation = isMobile ? height / 4 : height / 2),
```
Comment (chrishutchinson) [inline timeline-bubbles/D3/script.js]: What is going on here :mystery:

File: timeline-bubbles/index.html [hunk 2]
```diff
@@ -49,7 +49,7 @@
 
         <script type="text/javascript" src="https://unpkg.com/d3@4.13"></script>
         <script type="text/javascript" src="https://unpkg.com/d3-jetpack"></script>
-        <script type="text/javascript" src="https://unpkg.com/d3-svg-annotation"></script>
+        <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/d3-annotation/2.2.5/d3-annotation.min.js"></script>
```
Comment (chrishutchinson) [inline timeline-bubbles/index.html]: I'm guessing `d3-annotation` isn't on `unpkg`?

Comment (basilesimon) [reply timeline-bubbles/index.html]: yep I don't believe so.

File: timeline-bubbles/D3/script.js (hunk 1 above)
Comment (basilesimon) [reply timeline-bubbles/D3/script.js]: Weirdest `prettier` formatting I've seen

**Pull request merged**